xmodule/capa_block.py

```diff
@@ -1751,9 +1789,13 @@ def submit_problem(self, data, override_time=False):
             # self.attempts refers to the number of attempts that did not
             # raise an error (0-based)
             self.attempts = self.attempts + 1
+
             self.lcp.done = True
+
```

Comment:
Can we not update things we're not modifying? So we don't add changes unrelated to the implementation like these new lines.